connect/mirror-client/src/main/java/org/apache/kafka/connect/mirror/MirrorClientConfig.java

```diff
@@ -59,6 +59,10 @@ public class MirrorClientConfig extends AbstractConfig {
     public static final String REPLICATION_POLICY_SEPARATOR_DEFAULT =
         DefaultReplicationPolicy.SEPARATOR_DEFAULT;
 
+    public static final String INTERNAL_TOPIC_SEPARATOR_ENABLED =  "replication.policy.internal.topic.separator.enabled";
+    private static final String INTERNAL_TOPIC_SEPARATOR_ENABLED_DOC = "whether or not to use replication.policy.separator to control internal topic names";
```

Comment:
We should be explicit here about why we've provided this property:
```suggestion
    private static final String INTERNAL_TOPIC_SEPARATOR_ENABLED_DOC =
            "Whether to use replication.policy.separator to control the names of topics used for checkpoints and offset syncs. " +
                    "By default, custom separators are used in these topic names; however, if upgrading MirrorMaker 2 from older versions " +
                    "that did not allow for these topic names to be customized, it may be necessary to set this property to 'false' in order " +
                    "to continue using the same names for those topics.";
```